components/specification/publish

```diff
@@ -346,41 +348,20 @@ createmainindex() {
 <h2>Table of contents</h2>
         <ol>
                 <li><a href="#intro">Introduction</a></li>
-                <li><a href="#history">History</a></li>
                 <li><a href="#schema.list">Schemas</a></li>
+                <li><a href="#history">History</a></li>
         </ol>
 </div>
 <div id="intro">
 <h2>Introduction</h2>
-<p>This document outlines the XML Schemas created by the Open Microscopy Environment Group. Detailed information on the schemas and future development plans are available at <a href="http://www.openmicroscopy.org/site/support/file-formats">http://www.openmicroscopy.org/site/support/file-formats</a>. Further information on the groups work is available at <a href="http://www.openmicroscopy.org/">http://www.openmicroscopy.org/</a>
+<p>This document outlines the XML Schemas created by the Open Microscopy Environment Group. Detailed information on the schemas and future development plans are available at <a href="http://www.openmicroscopy.org/site/support/ome-model/index.html">http://www.openmicroscopy.org/site/support/ome-model/index.html</a>. The main OME website is available at <a href="http://www.openmicroscopy.org/">http://www.openmicroscopy.org/</a>
```

Comment:
To aid review and editing, could we also wrap these lines to 72 columns please?

Do we need an explicit link to the main website here?  Looks like a legacy of ome-xml.org.